File: TV_Reciever.pas
Program TV_Transmitter;
Const 
	Stop = '-'; // -1 HARDCODE TO FIRST CHAR
	BlackScreen = '0,0,0';
Var 
    signal : string;
Begin;
    ReadLn (signal)
    IF (signal[1] <> Stop) Then
		WriteLn (signal);
	Else 
		WriteLn (BlackScreen);
end.

{ by Tovarisch Trunaev, 2026, Russia, Uray }

File: TV_Reciever.pl
#!/usr/bin/perl

use strict;
use warnings;

my $signal = <>;
chomp $signal;

# if ($signal ne '-1') HARDCODE
if (substr($signal, 0, 1) ne '-') {
	say $signal;
} else {
	say "0,0,0";
	say "No signal"; # sign
}

1;

# by Tovarisch Trunaev, 2026, Russia, Uray
